src/main/java/us/warframestat/moddetection/api/API.java

```diff
@@ -0,0 +1,69 @@
+package us.warframestat.moddetection.api;
+
+import org.apache.commons.io.FileUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.Arrays;
+import java.util.Objects;
+
+/**
+ * Entrypoint on the API.
+ */
+public class API {
+    public static final Logger LOGGER = LoggerFactory.getLogger(API.class);
+
+    public static File DATA = new File(System.getProperty("java.io.tmpdir"), "ModDetectionOpenCV");
```

Comment:
I just registered that this is public static. That is pretty problematic for it to be reset potentially whine it's being used

Should probably be private.